tests/integration/test_labeling_parameter_overrides.py

```diff
@@ -45,10 +45,42 @@ def test_labeling_parameter_overrides(consensus_project, initial_dataset,
         data = [(data_rows[2], "a_string", 3)]
         project.set_labeling_parameter_overrides(data)
     assert str(exc_info.value) == \
-        f"Priority must be an int. Found <class 'str'> for data_row {data_rows[2]}. Index: 0"
+           f"Priority must be an int. Found <class 'str'> for data_row {data_rows[2]}. Index: 0"
 
     with pytest.raises(TypeError) as exc_info:
         data = [(data_rows[2].uid, 1)]
         project.set_labeling_parameter_overrides(data)
     assert str(exc_info.value) == \
-        "data_row should be be of type DataRow. Found <class 'str'>. Index: 0"
+           "data_row should be be of type DataRow. Found <class 'str'>. Index: 0"
+
+
+def test_set_labeling_priority(consensus_project, initial_dataset, rand_gen,
+                               image_url):
+    project = consensus_project
+    dataset = initial_dataset
+
+    task = dataset.create_data_rows([{DataRow.row_data: image_url}] * 3)
```

Comment:
can we use a fixture to create data rows? Else, if the test fails, data rows are not deleted.